controller/space_template.go

```diff
@@ -127,11 +127,12 @@ func ConvertSpaceTemplate(appl application.Application, request *http.Request, s
 		Type: APISpaceTemplates,
 		ID:   &st.ID,
 		Attributes: &app.SpaceTemplateAttributes{
-			Name:        &st.Name,
-			CreatedAt:   &st.CreatedAt,
-			UpdatedAt:   &st.UpdatedAt,
-			Version:     &st.Version,
-			Description: st.Description,
+			Name:           &st.Name,
+			CreatedAt:      &st.CreatedAt,
+			UpdatedAt:      &st.UpdatedAt,
+			Version:        &st.Version,
+			Description:    st.Description,
+			IsBaseTemplate: ptr.Bool(st.ID == spacetemplate.SystemBaseTemplateID),
```

Comment:
We should have a test for this even though it might seem obvious what's being done here. But tests help to keep our system stable over time.

Reply:
Added tests.